packs/chef/actions/chef_install.py

```diff
@@ -20,9 +19,10 @@ class ChefInstaller(object):
         ('-d', '--download_path', {})
     ]
 
-    def install(self, install_method, options):
+    @staticmethod
+    def install(install_method, options):
         exit_code = 0
-        installer = getattr(ChefInstaller, install_method.capitalize())(options)
+        installer = getattr(sys.modules['lib'], install_method.capitalize())(options)
```

Comment:
In any case, I would prefer to have an explicit mapping instead of doing this unnecessary import hackery which makes code hard to understand.

We only support one installation method anyway so this is really an over-kill.

Something like this is better:

```python
INSTALLER_NAME_TO_CLASS_MAP = {
  'omnibus': 'OmniBus
}

...
installer = dict.get
....
```
And yeah, I know you didn't originally contribute this code, but if we are already touching the code, let's improve it :)